config/locales/en.yml

```
@@ -76,6 +76,9 @@ en:
     overwatch_2:
       title_html: <span class="text-overwatch-2">Overwatch 2</span> Beta compatible&nbsp;codes
       button: View
+    workshop_disabled:
+      title: The Workshop is currently disabled
+      content_html: With the release of Overwatch 2 the Workshop is not yet enabled. <strong class="text-white">Previously created codes can still be imported</strong>. Custom games without the Workshop are still available.
```

```suggestion
      content_html: The Workshop is not yet enabled in Overwatch 2. <strong class="text-white">Previously created codes can still be imported</strong>. Custom games without the Workshop are still available.
```
Just took another look at the wording of the banner, and I think this version reads a little better.